SqlConnect.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Class to to make connection with MySQL and perform SQL queries.
+ */
+class SqlConnect {
+  private $conn;
+
+  /**
+   * Constructor function.
+   *
+   * @param string $servername
+   *   Name of the server.
+   *
```

Comment:
No need of empty lines between multiple params, only one empty line is needed between params and return data.